go.mod

```diff
@@ -68,6 +69,7 @@ require (
 go 1.16
 
 replace (
+	github.com/cs3org/go-cs3apis => github.com/ishank011/go-cs3apis v0.0.0-20210715114809-34729f68a479
```

Comment:
@ishank011 @labkode this made it to master, could you please take care to remove it?

Reply:
@wkloucek yep, created a PR